File: apps/web/public/bridging/Day3/C/arrays_and_loops.c
/*
 * Day 3 – Arrays and loops (C)
 * Creating arrays, iterating with for loop.
 * Compile: gcc -o arrays_and_loops arrays_and_loops.c
 * Run:     ./arrays_and_loops
 */
#include <stdio.h>

int main(void) {
    int numbers[] = { 10, 20, 30, 40, 50 };
    int n = sizeof(numbers) / sizeof(numbers[0]);

    printf("Numbers: ");
    for (int i = 0; i < n; i++) {
        printf("%d ", numbers[i]);
    }
    printf("\n");

    const char *names[] = { "Alice", "Bob", "Carol" };
    int num_names = 3;
    for (int i = 0; i < num_names; i++) {
        printf("Hello, %s\n", names[i]);
    }

    return 0;
}

File: apps/web/public/bridging/Day3/Java/ArraysAndLoops.java
/**
 * Day 3 – Arrays and loops (Java)
 * Creating arrays, iterating with for-each and for loop.
 * Compile: javac ArraysAndLoops.java
 * Run:     java ArraysAndLoops
 */
public class ArraysAndLoops {

    public static void main(String[] args) {
        // Create an array of integers
        int[] numbers = { 10, 20, 30, 40, 50 };

        // Loop with index
        System.out.print("Numbers: ");
        for (int i = 0; i < numbers.length; i++) {
            System.out.print(numbers[i] + " ");
        }
        System.out.println();

        // For-each loop
        System.out.print("Again:   ");
        for (int n : numbers) {
            System.out.print(n + " ");
        }
        System.out.println();

        // String array
        String[] names = { "Alice", "Bob", "Carol" };
        for (String name : names) {
            System.out.println("Hello, " + name);
        }
    }
}
